helm/aqua-app-enforcer/templates/image-pull-secret.yaml

```
@@ -6,7 +6,7 @@ metadata:
   name: {{ .Release.Name }}-registry-secret
   labels:
     app: {{ .Release.Name }}
-    chart: "{{ include "scanner.chartVersion" . }}"
+    chart: "{{ include "chartVersion" . }}"
```

This is generated from the helpers file: https://github.com/giantswarm/aqua-app/blob/sync_with_GS_branch/helm/aqua-app-scanner/templates/_helpers.tpl#L38

```suggestion
    chart: "{{ include "scanner.chartVersion" . }}"
```

---

Yea but that's the helper for the scanner. The one for enforcer is here: https://github.com/giantswarm/aqua-app/blob/9e09f63234049ef11182792afb5be5822b5e4aa1/helm/aqua-app-enforcer/templates/_helpers.tpl#L26 right?

---

oh yes my bad :facepalm: